src/css/media-queries.scss

```diff
@@ -0,0 +1,147 @@
+// @mixin iphone-size {
```

Comment:
Nice! I hadn't seen putting media queries in a separate file before. Makes a lot of sense, definitely stealing this!